rust/config/mainnet_config.json

```diff
@@ -19,15 +19,15 @@
       "chainId": 888888888,
       "displayName": "Ancient8",
       "domainId": 888888888,
-      "domainRoutingIsm": "0xB6F0f1267B01C27326F61a4B4fe2c73751802685",
+      "domainRoutingIsm": "0x477145b11E1a71fEb658d96A0E27F19495121504",
```

Comment:
I assume these are from the default ISM updates?

Reply:
yeah it's from a registry update